### PR TITLE
Fix #14214

### LINKED ISSUE #14214: [NGE] Add Export to .glb button
It would be very useful to have an export to .glb button in NGE. This button could be helpful in scenarios where a user needs to take geometry out of NGE and into a DCC tool, either for further creation or debugging purproses.

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Review by sebavan — APPROVED** [review]

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14216/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14216/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14216/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14216/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14216/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14216/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14216/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14216/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14216/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Review by Popov72 — APPROVED** [review]

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14216/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14216/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by carolhmj** [comment]
Pushed a fix to linting! 😄

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

File: packages/tools/nodeGeometryEditor/webpack.config.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = (env) => {
             extensions: [".js", ".ts", ".tsx", ".scss", "*.svg"],
             alias: {
                 "shared-ui-components": path.resolve("../../dev/sharedUiComponents/src"),
+                "serializers": path.resolve("../../dev/serializers/src")
```
Comment (RaananW) [inline packages/tools/nodeGeometryEditor/webpack.config.js:19]:
```suggestion
                "serializers": path.resolve("../../dev/serializers/dist")
```

**Pull request merged**